README.md

```diff
@@ -139,6 +139,7 @@ Styles are not added on `import/require()`, but instead on call to `use`/`ref`.
 |**`transform`** |`{Function}`|`false`|Transform/Conditionally load CSS by passing a transform/condition function|
 |**`insertAt`**|`{String}`|`bottom`|Inserts `<style></style>` at the given position|
 |**`insertInto`**|`{String}`|`<head>`|Inserts `<style></style>` into the given position|
+|**`insertBefore`**|`{String}`|``|A query string specifying an element to insert new `<style>` before, required when `insertAt="before"`|

```

Comment:
Yeah I wasn't completely sure the best way to approach this. The logic for `insertBefore` could also be triggered without requiring anything special in `insertAt`. In other words, if `insertBefore` has a non-empty string that behavior pre-empt the normal insertion logic triggered by `insertAt`.

I steered away from that because it seemed potentially confusing to have `insertBefore` pre-empt and essentially override the behavior of `insertAt`.

I'm also fine with making `insertAt` an object, but that becomes a breaking change. If you think that's best I can go with it, but it seems kind of heavy-handed for a small feature.